students/aravichander/lesson07/Mailroom_unittest.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from Mailroom_lesson07 import createlist,getlist,sumdonor
+
+class MyFuncTestCase(unittest.TestCase):
+	
+	def test_createlist(self):
+		createlist()
+		output = getlist()
+		self.assertEqual(output["Bill Gates"],[10000,12000]) 
+		self.assertTrue(output["Bill Gates"] == [10000,12000]) 
```

Comment:
If you are using `unittest`, you want to use the custom asserts (i.e. `assertEqual`) whenever possible. Doing an `assertTrue` with a `==` is exactly the same, but with less useful output on a failure.